app/views/early_years_payment/start/claims/email_address.html.erb

```diff
@@ -0,0 +1,27 @@
+<% content_for(:page_title, page_title(@form.t(:question), journey: current_journey_routing_name, show_error: @form.errors.any?)) %>
+
+<% @backlink_path = landing_page_path %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with model: @form, url: claim_path(current_journey_routing_name), method: :patch, builder: GOVUKDesignSystemFormBuilder::FormBuilder, html: { novalidate: false } do |f| %>
+      <%= f.govuk_error_summary %>
+
+      <h1 class="govuk-heading-l">
+        Enter your email address
+      </h1>
+
+      <p class="govuk-body">
+        You must use the email address previously provided to your local authority to access this service.
+      </p>
+
+      <%= f.govuk_text_field :email_address,
```

Comment:
we're not using it elsewhere but since we are now using govuk form builder maybe we can start using `govuk_email_field`